neo.UnitTests/UT_Consensus.cs

```diff
@@ -47,10 +47,10 @@ public void ConsensusService_Primary_Sends_PrepareRequest_After_OnStart()
             int timeIndex = 0;
             var timeValues = new[] {
               //new DateTime(1968, 06, 01, 0, 0, 15, DateTimeKind.Utc), // For tests here
-              new DateTime(1968, 06, 01, 0, 0, 1, DateTimeKind.Utc),  // For receiving block
-              new DateTime(1968, 06, 01, 0, 0, (int) Blockchain.MillisecondsPerBlock / 1000, DateTimeKind.Utc), // For Initialize
-              new DateTime(1968, 06, 01, 0, 0, 15, DateTimeKind.Utc), // unused
-              new DateTime(1968, 06, 01, 0, 0, 15, DateTimeKind.Utc)  // unused
+              new DateTime(1980, 06, 01, 0, 0, 1, 001, DateTimeKind.Utc),  // For receiving block
```

Comment:
Why 1980? Did you mean 1970? Unix 0 timestamp = 01/01/1970 00:00:00 UTC 😅

Reply:
Here it is ok, just on helper we keep the init in 1970